vowpalwabbit/core/src/reductions/cb/details/large_action/one_pass_svd_impl.cc

```diff
@@ -77,8 +96,15 @@ class AO_triplet_constructor
   uint64_t _column_index;
   uint64_t _seed;
   float& _final_dot_product;
+  // index mapped used to select sparsity or not from value map
+  constexpr static std::array<size_t, 2> index_map = {0, 2};
+  constexpr static std::array<float, 4> value_map = {0.f, 0.f, 1.f, -1.f};
```

Comment:
```suggestion
  constexpr static const std::array<size_t, 2> INDEX_MAP = {0, 2};
  constexpr static const std::array<float, 4> VALUE_MAP = {0.f, 0.f, 1.f, -1.f};
```

Reply:
aren't all constexpr variables already const?

Reply:
Yeah I think you're right, can remove the cosnt

Reply:
capitalized